README.md

```diff
@@ -405,23 +405,23 @@ Your contributions and suggestions are heartily♥ welcome. (✿◕‿◕). Plea
+* [Bugtraq (BID)](http://www.securityfocus.com/bid/) - Software security bug identification database compiled from submissions to the SecurityFocus mailing list and other sources, operated by Symantec, Inc.
 * [Exploit-DB](https://www.exploit-db.com/) - Non-profit project hosting exploits for software vulnerabilities, provided as a public service by Offensive Security.
 * [Microsoft Security Bulletins](https://technet.microsoft.com/en-us/security/bulletins#sec_search) - Announcements of security issues discovered in Microsoft software, published by the Microsoft Security Response Center (MSRC).
 * [Microsoft Security Advisories](https://technet.microsoft.com/en-us/security/advisories#APUMA) - Archive of security advisories impacting Microsoft software.
 * [Mozilla Foundation Security Advisories](https://www.mozilla.org/security/advisories/) - Archive of security advisories impacting Mozilla software, including the Firefox Web Browser.
 * [Packet Storm](https://packetstormsecurity.com/files/) - Compendium of exploits, advisories, tools, and other security-related resources aggregated from across the industry.
-* [CXSecurity](https://cxsecurity.com/) - Archive of published CVE and Bugtraq software vulnerabilities cross-referenced with a a Google dork database for discovering the listed vulnerability.
+* [CXSecurity](https://cxsecurity.com/) - Archive of published CVE and Bugtraq software vulnerabilities cross-referenced with a Google dork database for discovering the listed vulnerability.
 * [SecuriTeam](http://www.securiteam.com/) - Independent source of software vulnerability information.
 * [Vulnerability Lab](https://www.vulnerability-lab.com/) - Open forum for security advisories organized by category of exploit target.
 * [Zero Day Initiative](http://zerodayinitiative.com/advisories/published/) - Bug bounty program with publicly accessible archive of published security advisories, operated by TippingPoint.
-* [Vulners](https://vulners.com) - Security database of software vulnerabilities.
-* [Inj3ct0r](https://www.0day.today) ([Onion service](http://mvfjfugdwgc5uwho.onion/)) - Exploit marketplace and vulnerability information aggregator.
-* [Open Source Vulnerability Database (OSVDB)](https://osvdb.org/) - Historical archive of security vulnerabilities in computerized equipment, no longer adding to its vulnerability database as of April, 2016.
+* [Vulners](https://vulners.com/) - Security database of software vulnerabilities.
+* [Inj3ct0r](https://www.0day.today/) ([Onion service](http://mvfjfugdwgc5uwho.onion/)) - Exploit marketplace and vulnerability information aggregator.
+* [Open Source Vulnerability Database (OSVDB)](https://osvdb.org/) - Historical archive of security vulnerabilities in computerized equipment, no longer adding to its vulnerability database as of April, 2016. Continued by [Risk Based Security](https://vulndb.cyberriskanalytics.com/) as a commercial VDB.
```

Comment:
Adding the link to Risk Based Security's VulnDB would make it the first addition of a commercial vulnerability database to this list. I took a look at the VulnDB website but could not find any obvious way for someone who did not pay money for access to it to glean any information from it. In other words, it look like its information is only available to paying subscribers. Am I misunderstanding?

That kind of walling-off information does not feel Awesome to me, especially when so many other freely accessible databases are listed; this list's VulnDB section is not really in dire need for expansion.

Also, @attritionorg lists `riskbasedsecurity.com` as your profile URL. That URL in turn links directly to `vulndb.cyberriskanalytics.com` as a commercial service offering. I've got nothing against self-promotion per se, but this addition feels more like an advertisement to me if the VulnDB is indeed all or mostly sealed off and available to paying customers only. Most if not all of the other "commercial" items in this list across all categories at least offer a free version, so they at least have the possibility of being immediately useful to someone, even if their Awesomeness can be argued.